salt/modules/glanceng.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+'''
+Glance module for interacting with OpenStack Glance
+
+.. versionadded:: Oxygen
+
+:depends:shade
+
+Example configuration
+
+.. code-block:: yaml
+    glance:
+      cloud: default
+
+.. code-block:: yaml
+    glance:
+      auth:
+        username: admin
+        password: password123
+        user_domain_name: mydomain
+        project_name: myproject
+        project_domain_name: myproject
+        auth_url: https://example.org:5000/v3
+      identity_api_version: 3
+'''
+
+from __future__ import absolute_import
+
+import salt.utils
```

Comment:
We should not be importing `salt.utils` anymore in develop.

Reply:
this was already commented on above. @gtmanfred what should I be doing here?